requirements.txt

```diff
@@ -43,7 +43,7 @@ MarkupSafe==2.1.3
 mdurl~=0.1.2
 mock~=5.0.2
 packaging~=23.1
-Pillow==10.3.0
+Pillow==10.4.0
```

Comment:
It looks like the Pillow fix for 3.13 has been implemented: https://github.com/pypa/setuptools/issues/4415